src/SDK/Language/Flutter.php

```diff
@@ -223,6 +223,12 @@ public function getFiles()
                 'template'      => 'flutter/lib/src/service.dart.twig',
                 'minify'        => false,
             ],
+            [
+                'scope'         => 'default',
+                'destination'   => '/lib/src/models/model_base.dart',
```

Comment:
make this one `lib/src/models/model.dart`, we only made the template `model_base.dart.twig` because we already have `model.dart.twig` for other models generation
```suggestion
                'destination'   => '/lib/src/models/model.dart',
```